apps/accounts/urls.py

```
@@ -14,5 +14,6 @@
     path('profile', UserProfilesView.as_view(), name='user-profile'),
 
     path('detail', UserDetailsView.as_view(), name='user-detail'),
+    path('detail/<uuid:uuid>', WithUUIDView.as_view(), name='user-detail-uuid'),
```

Ruff

`WithUUIDView` may be undefined, or defined from star imports

(F405)